### ISSUE TITLE
AWS::Budgets::Budget BudgetType allowed value

### ISSUE THREAD

**Issue by IvanBlanquez** [aws-cloudformation/cfn-python-lint#1642]
*cfn-lint version* 

0.34.1

*Description of issue.*
This version reports an error when you use some of BudgetType allowed values based on [AWS CloudFormation user guide](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-budgets-budget-budgetdata.html#cfn-budgets-budget-budgetdata-budgettype). You can find an example below:

For code:
```
 SPBudget:
    Type: "AWS::Budgets::Budget"
    Properties:
      Budget:
        BudgetName: !Ref BudgetSPName
        TimeUnit: MONTHLY
        BudgetType: SAVINGS_PLANS_UTILIZATION
      NotificationsWithSubscribers:
        - Notification:
            NotificationType: ACTUAL
            ComparisonOperator: LESS_THAN
            Threshold: 85
            ThresholdType: PERCENTAGE
          Subscribers:
            - SubscriptionType: SNS
              Address: !Sub "arn:aws:sns:${AWS::Region}:${AWS::AccountId}:${TopicName}"
```

cfn-lint reports the following error but assigned value is a valid value:
```
[cfn-lint] E3030: You must specify a valid value for BudgetType (SAVINGS_PLANS_UTILIZATION).
Valid values are ["COST", "RI_COVERAGE", "RI_UTILIZATION", "USAGE"]
```

**Comment by PatMyron** [comment]
Thanks for the report. Should be fixed for the next upcoming release

Can workaround with [resource-level ignores](https://github.com/aws-cloudformation/cfn-python-lint#resource-based-metadata) until then:

```yaml
  SPBudget:
    Type: AWS::Budgets::Budget
    Metadata:
      cfn-lint:
        config:
          ignore_checks:
          - E3030
```
